server/controllers/finance/reports/financial.employee.js

```diff
@@ -42,22 +45,34 @@ function build(req, res, next) {
   }
 
   const data = {};
+  const sql = `
+    SELECT BUID(p.debtor_uuid) as debtor_uuid
+    FROM patient p, employee em
```

Comment:
Everywhere else we use `FROM patient p JOIN employee em ON p.uuid = em.xxx`.  In order to make it easier for new developers, could you use this syntax here?